pom.xml

```diff
@@ -1860,6 +1860,11 @@
                 <artifactId>james-server-webadmin-jmap</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>${james.groupId}</groupId>
+                <artifactId>james-server-webadmin-mail-over-web</artifactId>
```

Comment:
I personally like that separated module

Reply:
What are the pros of this separate module?

I see it as closely related to the mail queue, hence I wonder what we gain from separating it.

Reply:
From a user perspective, mailqueue is a technical detail, what matters to them/me is sending an email to some people. 

So having a component that allow to send emails to people looks good.

Cons:
* more module

Pros:
* easier to understand / find / document
* well separated features
* easier to audit dependencies